e2e/test/PnpServiceTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.Azure.Devices.Client;
+using Microsoft.Azure.Devices.Shared;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Azure.Devices.E2ETests
+{
+    [TestClass]
+    [TestCategory("E2E")]
+    [TestCategory("IoTHub")]
+    [TestCategory("PlugAndPlay")]
+    public class PnpServiceTests
+    {
+        private const string DevicePrefix = "plugAndPlayDevice";
+        private const string ModulePrefix = "plugAndPlayModule";
+        private const string TestModelId = "dtmi:com:example:testModel;1";
+
+        [TestMethod]
+        public async Task DeviceTwin_Contains_ModelId()
+        {
+            // Setup
+
+            // Create a device.
+            TestDevice testDevice = await TestDevice.GetTestDeviceAsync(DevicePrefix).ConfigureAwait(false);
```

Comment:
Since these tests runs under a .net core/ netfm console app, we shouldn't be requiring these `.ConfigureAwait(false)` statements here either, right?

Reply:
We don't need it but as this is not a sample that we will share, I am adding it for consistency. We've followed that in other E2E tests as well.